api/src/controllers/search.js

```diff
@@ -4,6 +4,7 @@ var mongoose = require('mongoose');
 var Actions = require('../utils/actions');
```

Comment:
Apparently you cant comment on lines that haven't changed.  So this is for line 2 above.

For logging, we can use the logger.js class I set up.  Adds some extra formatting and prepends the name you pass in to the logs so they read nicer.

```
const defaultLog = require('../utils/logger')('search');
```

Not an issue for this PR, but just in general for future stuff.